integration_test/third_party_apps_data/applications/dcgm/centos_rhel/install

```diff
@@ -1,7 +1,23 @@
 set -e
 source /etc/os-release
 KERNEL_VERSION=`uname -r`
-sudo yum install -y kernel-devel-${KERNEL_VERSION} pciutils gcc make wget yum-utils 
+
+# sudo yum install -y kernel-devel-${KERNEL_VERSION} pciutils gcc make wget yum-utils 
+# TODO(b/312455563): Temporary fix for Rocky Linux 9.2/8.8; uncomment the above line and remove this following section once the image has been updated to Rocky Linux 9.3/8.9
```

Comment:
Will we leave this bug open to track the update to the image? If not, I'd create a bug for it and add a link to that instead.

Reply:
Done!